prow/config/inrepoconfig.go

```diff
@@ -59,10 +59,7 @@ var inrepoconfigRepoOpts = git.RepoOpts{
 	SparseCheckoutDirs: []string{inRepoConfigDirName, inRepoConfigFileName},
 	// The sparse checkout would avoid creating another copy of Git objects
 	// from the mirror clone into the secondary clone.
-	ShareObjectsWithSourceRepo: true,
-	// Disable fetching tag objects, because they are redundant and useless (we
```

Comment:
Why is this removed?

Reply:
It was renamed. See https://github.com/kubernetes/test-infra/pull/30428/commits/6b1f69dac3ad9c4448d023c13928c0161be2f6c7.

As a rule of thumb, if there are multiple commits in a PR, you should review each commit in isolation. Then you'll get to see which changes belong together (which is not apparent in the combined diff).